cookbooks/bcpc/recipes/graphite.rb

```diff
@@ -27,13 +27,13 @@
 # These data bags and vault items are pre-populated at compile time by
 # the bcpc::mysql_data_bags recipe.
 #
-root_user = get_config!('mysql-root-user')
-root_password = get_config!('password', 'mysql-root', 'os')
+# root_user = get_config!('mysql-root-user')
+# root_password = get_config!('password', 'mysql-root', 'os')
```

Comment:
If the root user/pass are no longer necessary, can we just remove these two lines?